pages/instructorAssessmentSettings/instructorAssessmentSettings.ejs

```diff
@@ -97,9 +102,16 @@
                       <td class="form-inline align-middle">
                         <span class="input-group">
                           <span readonly class="form-control form-control-sm"><%= studentLink %></span>
-                          <button type="button" class="input-group-prepend btn btn-sm btn-outline-secondary btn-copy" data-clipboard-text="<%= studentLink %>">
+                          <button type="button" class="input-group-append btn btn-sm btn-outline-secondary btn-copy" data-clipboard-text="<%= studentLink %>">
                             <i class="far fa-clipboard"></i>
                           </button>
+                          <button type="button" title="Student Link QR Code"
+                                  class="input-group-append btn btn-sm btn-outline-secondary js-student-link-qrcode-button">
+                            <i class="fas fa-qrcode"></i>
+                          </a>
```

Comment:
Two changes:

- Uses the correct markup from https://getbootstrap.com/docs/4.6/components/input-group/#button-addons
- Fixes the closing tag

```suggestion
                          <div class="input-group-append">
                            <button type="button" class="btn btn-sm btn-outline-secondary btn-copy" data-clipboard-text="<%= studentLink %>">
                              <i class="far fa-clipboard"></i>
                            </button>
                            <button type="button" title="Student Link QR Code"
                                  class="btn btn-sm btn-outline-secondary js-student-link-qrcode-button">
                              <i class="fas fa-qrcode"></i>
                            </button>
                          </div>
```

Can you make the same changes to the other page too?

Reply:
Done.